modules/dynamic-provisioning-vsphere-definition.adoc

```diff
@@ -13,8 +13,11 @@ apiVersion: storage.k8s.io/v1
 metadata:
   name: <storage-class-name> <1>
 provisioner: csi.vsphere.vmware.com <2>
+parameters:
+  csi.storage.k8s.io/fstype: xfs   <3>
 ----
 <1> Name of the storage class. The persistent volume claim uses this storage class for provisioning the associated persistent volumes.
 <2> For more information about using VMware vSphere CSI with {product-title},
+<3>  XFS or EXT4 are supported filesystem type for vSphere CSI Driver. 
```

Comment:
Change to: "The vSphere CSI driver supports all of the file systems supported by the underlying Red Hat Core operating system release, including XFS and Ext4."